pyproject.toml

```diff
@@ -14,8 +14,12 @@ thrift = "^0.16.0"
 pandas = "^1.3.0"
 pyarrow = "^9.0.0"
```

Comment:
The Python 3.11 compatible wheels are for pyarrow versions >=10.0.1, so this line needs to change to make this PR work.

You could try using `pyarrow = ">=9.0.0"` or maybe better

```
pyarrow = [
    {version = ">=9.0.0", python = ">=3.7,<3.11"},
    {version = ">=10.0.1", python = ">=3.11"}
]
```

Reply:
Hey, as @potiuk said in #59, the pyarrow wheels for Python 3.11 are now out.

I could install this package successfully from a local git repo clone by slacking off the above pyarrow requirement. Please see above comment